packages/scanner/src/cli/scan/watchScan.ts

```diff
@@ -118,7 +118,8 @@ export class Watcher {
     this.queue.push(mtimePath);
   }
 
-  protected async scan(mtimePath: string): Promise<void> {
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  protected async scan(mtimePath: string, callback?: Function): Promise<void> {
```

Comment:
Good catch. We're compiling down to ES2015 so the native async on this method is replaced with a polyfill. I'm surprised this was only not working on Windows.

My only nit-pick here is the use of `Function` over `() => void | Promise<void>` or a similar typedef. ESLint doesn't like `Function` it because it'll accept any type of function or class.

Reply:
Okee dokee, I'll change it